src/deadline/max_adaptor/MaxClient/render_element_manager.py

```diff
@@ -351,15 +382,30 @@ def _configure_vray_settings(
 
             if vfb_control or split_buffer:
                 self.logger.info(
-                    f"Configuring V-Ray settings - VFB Control: {vfb_control}, Split Buffer: {split_buffer}"
+                    f"Configuring V-Ray settings for '{self.current_renderer}' - VFB Control: {vfb_control}, Split Buffer: {split_buffer}"
                 )
+                if split_buffer:
+                    if self.output_file_path and self.output_file_name:
+                        self.logger.info(
+                            f"Split buffer output: {self.output_file_path}/{self.output_file_name}"
+                        )
```

Comment:
nit: should we do `os.path.join` here? Especially since 3ds Max usually runs on Windows 😄 
```suggestion
                        self.logger.info(
                            f"Split buffer output: {os.path.join(self.output_file_path, self.output_file_name)}"
                        )
```

Reply:
Good point - let me change that since the comment can be confusing for the reader.